### PR TITLE
UpdateAccount fix

### PR DESCRIPTION
**Pull request by SammDev1**
Hi Alex,

This is the change needed to update the Stripe Account. 

Could you please revise the changes.

### CODE REVIEW

File: src/Message/UpdateAccountRequest.php [hunk 1]
```diff
@@ -9,13 +9,196 @@
...
+    public function setEmail($value)
+    {
+        return $this->setParameter('email', $value);
+    }
+
+    /**
+     * Get the account's metadata.     *
+     * @return string
+     */
+    public function getMetadata()
```
Comment (lembdev) [inline src/Message/UpdateAccountRequest.php]: This method allready defined in [Omnipay\Stripe\Message\AbstractRequest](https://github.com/thephpleague/omnipay-stripe/blob/master/src/Message/AbstractRequest.php#L104)

File: src/Message/UpdateAccountRequest.php [hunk 2]
```diff
@@ -9,13 +9,196 @@
...
+    {
+        return $this->getParameter('metadata');
+    }
+
+    /**
+     * Sets the account's metadata.     *
+     * @param string $value     *
+     * @return UpdateAccountRequest provides a fluent interface.
+     */
+    public function setMetadata($value)
```
Comment (lembdev) [inline src/Message/UpdateAccountRequest.php]: This method allready defined in [Omnipay\Stripe\Message\AbstractRequest](https://github.com/thephpleague/omnipay-stripe/blob/master/src/Message/AbstractRequest.php#L109)

File: src/Message/UpdateAccountRequest.php [hunk 3]
```diff
@@ -9,13 +9,196 @@
...
      * @inheritdoc
      */
     public function getData()
     {
         $this->validate('accountReference');
 
-        return $this->parameters->all();
+        $data = [];
+
+        //$data['type'] = $this->getType();
```
Comment (lembdev) [inline src/Message/UpdateAccountRequest.php]: The commented code should be removed

File: src/Message/UpdateAccountRequest.php [hunk 4]
```diff
@@ -9,13 +9,196 @@
 class UpdateAccountRequest extends AbstractRequest
 {
     /**
+     * Get the account's type.     *
```
Comment (lembdev) [inline src/Message/UpdateAccountRequest.php]: PHPDoc does not use the `*` character at the end of the line

**Comment by SammDev1** [comment]
Hi Alex. Changes done. Do I need to create another pull request?

**Pull request merged**